### PR TITLE
python3Packages.dinghy: init at 0.13.2

### PR DESCRIPTION
**Pull request by veehaitch**
###### Description of changes

Adds the `dinghy` Python package.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Trundle — APPROVED** [review]

**Comment by viraptor** [comment]
Result of `nixpkgs-review pr 186763` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.dinghy</li>
    <li>python39Packages.dinghy</li>
  </ul>
</details>

File: pkgs/development/python-modules/dinghy/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    jinja2
+    pyyaml
+  ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "dinghy.cli" ];
+
+  meta = with lib; {
+    description = "Dinghy daily digest tool";
```
Comment (risicle) [inline pkgs/development/python-modules/dinghy/default.nix:40]: Putting the word "github" in here would make it clearer for the uninitiated.

Comment (veehaitch) [reply pkgs/development/python-modules/dinghy/default.nix:40]: I just copied the description from here: https://github.com/nedbat/dinghy/blob/1ab5e4269973af2514d511c60790eb6c384f49d7/setup.cfg#L6

Nevertheless, I agree that the description isn't very good:

```suggestion
    description = "A GitHub activity digest tool";
```

Comment (veehaitch) [reply pkgs/development/python-modules/dinghy/default.nix]: Done!

**Review by Trundle — APPROVED** [review]

File: pkgs/development/python-modules/dinghy/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+, pytestCheckHook
+, aiofiles
+, aiohttp
+, click-log
+, emoji
+, glom
+, jinja2
+, pyyaml
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/dinghy/default.nix:13]:
```suggestion
}:

buildPythonPackage rec {
```

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.dinghy

**Review by Trundle — APPROVED** [review]

**Pull request merged**